agenda-webapps/src/main/webapp/vue-app/agenda-admin-settings/components/AgendaAdminConnectorSettings.vue

```diff
@@ -49,7 +49,7 @@
                   <v-slide-x-reverse-transition mode="out-in">
                     <i
                       :key="`icon-${props.item.editing}`"
-                      :class="props.item.editing ? 'uiIcon uiIconTick clickable success--text mt-1' : 'uiIcon uiIconEdit clickable primary--text mt-1'"
+                      :class="props.item.isExchange ? 'd-none' : props.item.editing ? 'uiIcon uiIconTick clickable success--text mt-1' : 'uiIcon uiIconEdit clickable primary--text mt-1'"
```

Comment:
Why using extensible component plugins if you want to embed the behavior for each connector inside the UI.
IMO, we don't need an extensible framework anymore, we should add isGoogle, isOffice365, isExchange ... The already made work for making this as an extensible framework shouldn't have pieces of plugins. But instead, we should improve the framework to allow different use cases.

Reply:
> Why using extensible component plugins if you want to embed the behavior for each connector inside the UI. IMO, we don't need an extensible framework anymore, we should add isGoogle, isOffice365, isExchange ... The already made work for making this as an extensible framework shouldn't have pieces of plugins. But instead, we should improve the framework to allow different use cases.

this is just a draft to synchronize the work with @abenmbarek !

Reply:
> You can base on another prop called isOauth for example which allows to decide if the api key is editable or not. This will keep the framework extensible, avoid mentioning exchange in the agenda codebase and prepare the transfert to the agenda-connectors addon codebase

@boubaker is that solution answers to your concern ?